pom.xml

```diff
@@ -51,6 +51,10 @@
                 <version>${kotlin.version}</version>
                 <configuration>
                     <jvmTarget>11</jvmTarget>
+                    <sourceDirs>
+                        <source>src/main/kotlin</source>
+                        <source>src/main/java</source>
```

Comment:
project does not have java and there is no plans to add it. so this need to be removed